bedrock/firefox/templates/firefox/os/devices.html

```diff
@@ -2784,6 +2790,65 @@ <h3 class="device-title">{{ _('Panasonic CX700') }}</h3>
             </div>{#--/.pager-content--#}
           </div>{#--/.container--#}
         </section>{#--/#panasonic_cx700--#}
+
+        <section class="device-detail smartphone-detail" id="panasonic_dx900">
+          <div id="panasonic_dx900-pager" class="container pager pager-with-tabs pager-no-history">
+            <header class="device-detail-header">
+              <h3 class="device-title">{{ _('Panasonic DX900') }}</h3>
+
+              <ul class="pager-tabs">
+                <li><a href="#p1" data-label="Features">{{_('Features')}}</a></li>
+              </ul>
+
+              <button class="device-detail-close">{{ _('Close') }}</button>
+            </header>
+
+            <div class="pager-content">
+              <div class="device-details features pager-page default-page" id="panasonic_dx900-features">
+                <div class="feature-group image wide">
+                  <img src="{{ static('img/firefox/os/devices/tvs/panasonic-dx900.png') }}" alt="{{ _('Panasonic DX900') }}">
+                </div>
+
+                <div class="feature-group stats">
+                  <ul>
+                    <li class="stat-general">
+                      <strong>{{ _('Picture Quality') }}</strong>
+                      {{ _('4K ULTRA HD TV') }}
+                      <br>
+                      {{ _('3,840(W) x 2,160(H)') }}
```

Comment:
I'd recommend reusing text from the CX700 instead of introducing a new yet almost identical string.